.github/workflows/test.yaml

```diff
@@ -26,4 +26,7 @@ jobs:
     - name: Install project
       run: |
         python -m pip install --upgrade pip
-        pip install "."
\ No newline at end of file
+        pip install "."
+    - name: Test Download
+      working-directory: ./src
+      run: python noisepy.py download --start 2019_02_01_00_00_00 --end 2019_02_01_02_00_00 --stations ARV --channels BHZ --inc_hours 1 --path .
```

Comment:
This downloads <300k worth of data. Is this sufficient for testing? Too much? too little?

Reply:
Maybe having a three-component downloaded? `--channels BHE,BHN,BHZ`. But the wildcard "?" should work for [obspy.client](https://docs.obspy.org/packages/autogen/obspy.clients.fdsn.client.Client.get_waveforms.html). I am not sure why the code says
```
default_chan_list = ["BHE","BHN","BHZ"]                                             
# channel if down_list=false (format like "HN?" not work here)
```

Reply:
Done!